pkg/resolution/resolver/cluster/resolver.go

```diff
@@ -101,6 +104,8 @@ func (r *Resolver) Resolve(ctx context.Context, origParams []pipelinev1beta1.Par
 	}
 
 	var data []byte
+	var uid string
+	groupVersion := "tekton.dev/v1beta1"
```

Comment:
We can probably just use `v1beta1.SchemeGroupVersion.String()` here.

Reply:
Sounds good. I changed it in this pr https://github.com/tektoncd/pipeline/pull/5705 because I am not sure if we really need to move this PR forward. FYI: The reason why this PR might not be moved forward is that resource in cluster is not in version control which is desired for slsa provenance. This implies that setting source value for in-cluster resources might be meaningless here.

PTAL that pr. Thank you!! cc @XinruZhang Thanks!